lib/tasks/upgrade.rake

```diff
@@ -143,11 +151,16 @@ namespace :upgrade do
     Rake::Task['upgrade:add_missing_token_permission_types'].execute
     orgs = Org.where(is_other: false).select(:id)
     orgs.each do |org|
-      org.grant_api!(TokenPermissionType.where(token_type: 'statistics'))
+      unless org.token_permission_types.include?(TokenPermissionType::STATISTICS)
+        org.grant_api!(TokenPermissionType.where(token_type: 'statistics'))
+      end
+      unless org.token_permission_types.include?(TokenPermissionType::PLANS)
+        org.grant_api!(TokenPermissionType.where(token_type: 'plans'))
+      end
     end
     users = User.joins(:perms).where(org_id: orgs).where(api_token: [nil, ''])
     users.each do |user|
-      if user.can_org_admin?
+      if user.can_org_admin? && user.api_token.blank?
```

Comment:
Thanks for updating this to be robust for re-running on data.

Do you think there's a use-case for converting some tasks like this one over to a utilities file rather than the upgrade namespace?